### ISSUE TITLE
test_pthread_c11_threads_proxied is flaky on Chromium CI

### ISSUE THREAD

**Issue by dschuff** [emscripten-core/emscripten#22617]
It seems to be primarily on Windows;
A recent example is [here](https://logs.chromium.org/logs/emscripten-releases/buildbucket/cr-buildbucket/8736138341013155377/+/u/Emscripten_testsuite__core0_/stdout) but most [recent failures](https://ci.chromium.org/ui/p/emscripten-releases/builders/ci/win) are this issue.

4 of the 50 most recent WIndows CI builds failed with this issue.

**Comment by dschuff** [comment]
Is it possible to set @flaky on just one config of a test that is parameterized? If not, I guess we could just mark the whole set as flaky?

**Comment by sbc100** [comment]
No, you can't mark just one variant as flaky.  I guess we just mark the whole thing but mention `_poxied` here in the bug report.
